plugins/provisioning-terraform-aws-core/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@amplication/plugin-provisioning-terraform-aws-core",
+  "version": "0.0.1",
+  "description": "Adds terraform code for provisioning Amazon Web Services core network infrastructure",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\""
+  },
+  "author": "Levi van Noort <levi@amplication.com>",
+  "license": "Apache-2.0",
+  "devDependencies": {
```

Comment:
Like in the gh-actions-ecs plugin, here too the lodash and code-gen-types should move from devDependencies to dependencies (they are used in your code in the plugin runtime).